uber/utils.py

```diff
@@ -285,3 +285,19 @@ def mount_site_sections(module_root):
     for section in sections:
         module = importlib.import_module(basename(module_root) + '.site_sections.' + section)
         setattr(Root, section, module.Root())
+
+
+def build_uber_absolute_url(relative_uber_page_url):
```

Comment:
Second the suggestion about the name change.  Dunno that we need the extra parameter though.